todo_app/app.py

```diff
@@ -1,60 +1,66 @@
-from flask import abort, Flask, render_template, request, redirect, session, url_for
-from todo_app.data.trello_api import Trello
-from todo_app.flask_config import Config
-from os import environ
-
-BOARD_KEY = environ.get('BOARD_KEY')
-TRELLO_KEY = environ.get('TRELLO_KEY')
-TRELLO_TOKEN = environ.get('TRELLO_TOKEN')
-
-app = Flask(__name__)
-app.config.from_object(Config)
-trello = Trello(BOARD_KEY, TRELLO_KEY, TRELLO_TOKEN)
-
-@app.route('/', methods=['POST', 'GET'])
-def index():
-    items = trello.get_all_cards()
-        
-    if request.method == 'GET':
-        items.sort(key=is_checked)
-        return render_template('index.html', items=items)
-    elif request.method == 'POST':
-        if new_item := request.form['field_name']:
-            trello.add_card(new_item)
-
-        if deleted_item_ids := [key[7:] for key in request.form.keys() if key.startswith('delete_')]:
-            print(deleted_item_ids)
-            for id in deleted_item_ids:
-                trello.delete_item(id)
-            return redirect('/')
+def create_app():
+    import flask
+    from flask import abort, Flask, render_template, request, redirect, session, url_for
+    from todo_app.data.trello_api import Trello
+    from todo_app.data.view_model import ViewModel
+    from todo_app.flask_config import Config
```

Comment:
Importing it here is one way around the config loading before .env in tests issue 😄

It should work fine here, but be aware that Python caches imports, so these are imported the first time you create_app, but the cached versions will be used thereafter. That can lead to confusing behaviour and is why imports are usually put at the top of a file.